introduction.md

```diff
@@ -0,0 +1 @@
+This is RajyaLakshmi02. This task is done to complete the tutorial episode2
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.

Reply:
I made the changes to introduction.md as specified